CMakeLists.txt

```diff
@@ -79,10 +79,6 @@ if ( EXIV2_ENABLE_EXTERNAL_XMP )
   set(EXIV2_ENABLE_XMP OFF)
 endif()
 
-if(BUILD_TESTING AND EXIV2_BUILD_UNIT_TESTS)
-  set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON) # Requires CMake 3.3.3
```

Comment:
Not sure why this was there to begin with... 🤷

Reply:
So as part of my meson work, I found brokenness with unit tests testing stuff that's not even compiled because of missing zlib and whatnot. That's all been fixed already. This smells like a hack to fix such problems.

Reply:
Ey hi! I introduced this some years ago because I wanted to test some classes which were not part of the public API. Since we were not exporting the symbols of those classes, when trying to compiling/link the unit tests I was getting some linking issues. 

If the CI jobs are not complaining probably this is not needed anymore ;)

Anyways it was a ugly hack, it is better to not have such kind of tricks in the CMake code.